File: Packages/MapBox/AnnotationObject.m
//
//  NewAnnotationObject.m
//  MapBoxTesting
//
//  Created by Dhvl's iMac on 23/08/17.
//  Copyright © 2017 Dhvl's iMac. All rights reserved.
//

#import "AnnotationObject.h"

@implementation AnnotationObject


- (id) initWithName:(NSString *)name latitude:(double)latitude longitude:(double)longitude identifier:(int)identifier fontColor:(NSString *)fontColor andBackgroundColor:(NSString *)bgColor {
    self = [super init];
    if (self) {
        self.name = name;
        self.latitude = latitude;
        self.longitude = longitude;
        self.identifier = identifier;
        self.fontColor = fontColor;
        self.bgColor = bgColor;
    }
    return self;
}

@end

File: Packages/MapBox/AnnotationObject.java
package com.apps.neonmapbox;

/**
 * Created by dhvlsimac on 18/09/17.
 */

public class AnnotationObject {

    public String name;
    public double latitude;
    public double longitude;
    public int identifier;
    public String icon;
    public String fontColor;
    public String bgColor;

    public AnnotationObject(String name, double lat, double lng, int identifier, String icon, String fontColor, String bgColor) {
        this.name = name;
        this.latitude = lat;
        this.longitude = lng;
        this.identifier = identifier;
        this.icon = icon;
        this.fontColor = fontColor;
        this.bgColor = bgColor;
    }

    public String getName() {
        return name;
    }

    public double getLatitude() {
        return latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public int getIdentifier() {
        return identifier;
    }

    public String getIcon() {
        return icon;
    }

    public String getFontColor() {
        return fontColor;
    }

    public String getBGColor() {
        return bgColor;
    }

}
